app/services/record_participant_declaration.rb

```diff
@@ -75,9 +75,15 @@ def user_profile
     end
   end
 
+  def declaration_type
+    if early_career_teacher? || mentor?
+      ParticipantDeclaration::ECF
+    end
+  end
```

Comment:
And otherwise it's `nil`? That doesn't look right...

Reply:
Current code only support ECF. We've got another ticket to add in NPQ and I want to refactor this when we add it.

Reply:
Alright, I wrote 'that doesn't look right...' cause I saw NPQ declarations and thought they would fit here :D